CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 ## bindhosts
-Systemless hosts for Apatch, KernelSU and Magisk
+A fully standalone implementation to systemlessly update the hosts file.
```

Comment:
this has to mirror the one in readme
try to also keep it short and concise, or dont do changes at all